litestar/handlers/websocket_handlers/listener.py

```diff
@@ -358,8 +358,16 @@ class WebsocketListener(ABC):
     type_encoders: A mapping of types to callables that transform them into types supported for serialization.
     """
 
-    def __init__(self) -> None:
-        self._handler = websocket_listener(
+    def __init__(self, owner: Router) -> None:
+        """Initialize a WebsocketListener instance.
+
+        Args:
+            owner: The :class:`Router <.routing.Router>` instance that owns this listener.
```

Comment:
```suggestion
            owner: The :class:`Router <litestar.routing.Router>` instance that owns this listener.
```

Reply:
Oh, I think I've done this from time to time :grimacing:

We should absolute reference?

Reply:
No, but you have to use the correct name.. Seems like I have corrected the wrong thing here :upside_down_face: 

It's `.router.Router`, not `.routing.Router` :grimacing:

Reply:
Oh cool, np then!